Geodata_CityGML/Geodata_CityGML_ontology.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix Geodata_CityGML: <https://github.com/GAIA-X4PLC-AAD/map-and-scenario-data/tools/ontologie_creator/ontologies/Geodata_CityGML_ontology/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix gax-core: <https://w3id.org/gaia-x/core#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+Geodata_CityGML: a owl:Ontology ;
+    rdfs:label "ontology definition for Geodata_CityGML"@en ;
+    dcterms:contributor "Maximilian Sindram (VCS)" ;
+    owl:versionInfo "0.1"^^xsd:float .
+
+Geodata_CityGML:Asset a owl:Class ;
```

Comment:
Asset should be renamed to GeoDataCityGml or GeodataCityGml

What is GML? Does everybody know this abbreviation? Should we not have the complete name here?